### ISSUE TITLE
Windows cannot find 'vim80\uninstall-gui.exe' and that file is absent from in recent packages

### ISSUE THREAD

**Issue by idbrii** [vim/vim-win32-installer#64]
When uninstalling an older version of vim8, the vim8.1 installer complains:

> Windows cannot find 'c:\apps\vim80\uninstall-gui.exe'.

Looking at the latest release, [gvim_8.1.0229_x64.zip](https://github.com/vim/vim-win32-installer/releases/download/v8.1.0229/gvim_8.1.0229_x64.zip) does not contain `uninstall-gui.exe`. It does contain `uninstall.exe`. Same for gvim_8.1.0229_x86.zip.

Did a rename occur? (I'm not sure if it ever existed. It's absent from my old 8.0.596 install too.)

**Comment by k-takata** [comment]
Did you install Vim 8.0 using the self installer .exe?
`uninstall-gui.exe` is automatically created by the self installer, so it is not contained in the zip packages.

Another possibility is that the file was mistakenly detected as a virus (like #53, etc.) and removed by the antivirus software.

**Comment by k-takata** [comment]
Not enough information. Closing.

**Comment by idbrii** [comment]
I'm fairly sure I've only used the zip archive packages on this machine (gvim_8.0.0596_x64.zip seems to be the first vim I installed) and not the "64-bit installer with the modern interface (MUI2)" or predecessor (i.e., gvim_8.1.0527_x64-mui2.exe).

When I use the zip archive packages, I use the contained install.exe. I assume that's not what you're asking about?

I don't understand why using the zip archive package's install.exe would depend on previously using the self-installer, so maybe I misunderstand your question?
